### PR TITLE
Add io package changes to 2201.2.0 release note

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose
$Subject

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md [hunk 1]
```diff
@@ -84,6 +84,10 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 #### New features
 
+##### `io` package
+
+- Added the support for datamapping in CSV read/write APIs.
```
Comment (niveathika) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:89]: Added support for datamapping between Ballerina Records and CSV in CSV read/write APIs.

Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:89]:
```suggestion
- Added the support for data mapping in CSV read/write APIs
```

Comment (sahanHe) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md]: updated in [5487afa](https://github.com/ballerina-platform/ballerina-release/pull/2091/commits/5487afa652eda8868da05e9e5fe3723497754972)

Comment (sahanHe) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md]: updated in [5487afa](https://github.com/ballerina-platform/ballerina-release/pull/2091/commits/5487afa652eda8868da05e9e5fe3723497754972)

**Review by niveathika — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**